internal/schema2/devices.go

```diff
@@ -39,4 +39,8 @@ type Devices struct {
 	FlexibleIov map[string]FlexibleIoDevice `json:"FlexibleIov,omitempty"`
 
 	SharedMemory *SharedMemoryConfiguration `json:"SharedMemory,omitempty"`
+
+	// TODO: This is pre-release support in schema 2.3. Need to add build number
+	// docs when a public build with this is out.
+	VirtualPci map[string]VirtualPciDevice `json:",omitempty"`
```

Comment:
Is this missing a json name?

Reply:
There's no need to include the JSON name if it's the same as the field name, right?

Reply:
(Of course in this case maybe the field name should be VirtualPCI so the JSON name would be necessary)